applications/nrf5340_audio/src/modules/audio_sync_timer.c

```diff
@@ -107,44 +107,41 @@ uint32_t audio_sync_timer_capture(void)
 	return timestamp_from_rtc_and_timer_get(tick, remainder_us);
 }
 
-uint32_t audio_sync_timer_capture_get(void)
+uint32_t audio_sync_timer_frame_start_capture_get(void)
 {
 	uint32_t cc_get_calls = 0;
 	uint32_t tick = 0;
 	static uint32_t prev_tick;
 	uint32_t remainder_us = 0;
-	static uint32_t prev_remainder_us;
 
-	/* This function is called too soon after I2S frame start may
-	 * result in values not yet being updated in the *_cc_get calls.
-	 * Ref: OCT-2585. To ensure new values are fetched, they are
-	 * read in a while-loop with a timeout.
+	/* This ISR is called before the I2S FRAMESTART event which does timer capture,
+	 * resulting in values not yet being updated in the *_cc_get calls.
+	 * To ensure new values are fetched, they are read in a while-loop with a timeout.
+	 * Depending on variables such as sample frequency, this delay varies.
+	 * For 16 kHz sampling rate is about 20 us. Less for 24 kHz and 48 kHz.
+	 * Ref: OCT-2585 and OCT-3368.
 	 */
 
 	do {
 		tick = nrf_rtc_cc_get(audio_sync_lf_timer_instance.p_reg,
 				      AUDIO_SYNC_LF_TIMER_I2S_FRAME_START_EVT_CAPTURE_CHANNEL);
 		cc_get_calls++;
 		if (cc_get_calls > CC_GET_CALLS_MAX) {
-			LOG_WRN("Unable to get new CC value");
+			LOG_ERR("Unable to get new tick from nrf_rtc_cc_get");
```

Comment:
Are you sure this should be an error? What are the consequences if the ticks are not gotten? Out of sync or total crash?